### DISCUSSION TITLE
How to listen to onReorderClick action complete in ArrayFieldTemplate

### DISCUSSION THREAD

**Discussion by andreaguru** [rjsf-team/react-jsonschema-form#4235] [Q&A]
Hi, i am using templates: ArrayFieldTemplate to create my custom template for Arrays.
I also want to apply a CSS animation when an array element has been moved down/up using onReorderClick, so that the user can see in a better way what happened. But I do not see any way to listen to this action, can you please help me?

Thanks a lot

**Comment by heath-freenome** [comment]
If you update the `ArrayFieldItemTemplate` you can intercept the `onReorderClick` there I believe
